lib/helpers/parseHeaders.js

```diff
@@ -2,6 +2,15 @@
 
 var utils = require('./../utils');
 
+// Headers whose duplicates are ignored by node
+// c.f. https://nodejs.org/api/http.html#http_message_headers
+var ignoreDuplicateOf = [
+  'age', 'authorization', 'content-length', 'content-type', 'etag',
+  'expires', 'from', 'host', 'if-modified-since', 'if-unmodified-since',
+  'last-modified', 'location', 'max-forwards', 'proxy-authorization',
+  'referer', 'retry-after', 'user-agent'
+];
```

Comment:
I'm not entirely sure this is _needed_, as almost all of these headers are only seen in requests, not responses, but it's probably better to match node's behavior.